### DISCUSSION TITLE
Anyone able to get iOS password autofill feature to work?

### DISCUSSION THREAD

**Discussion by charlieswing** [invertase/react-native-firebase#6776] [Q&A]
I am trying to implement [Password Autofill](https://developer.apple.com/documentation/security/password_autofill/) feature for the iOS version of my React Native application that uses Firebase however I cannot get this to autofill.

All documentation I've seen for implementing this feature for React Native is pretty straightfoward however when it comes to hosting the apple-app-site-association in Firebase, there is some kind of disconnect. Below are the steps I take to try to implement this feature:

Add textContentType to username and password fields:
```
<TextInput
  style={styles.input}
  placeholderTextColor={QuietCollectionColors.gray}
  onBlur={onBlur}
  onChangeText={value => {
   setUsername(value);
   onChange(value);
  }}
  value={value}
  placeholder="Email"
  autoCapitalize="none"
  textContentType="username"
  keyboardType="email-address"
/>
<TextInput
  style={styles.input}
  placeholderTextColor={QuietCollectionColors.gray}
  onBlur={onBlur}
  onChangeText={value => {
    setPassword(value);
    onChange(value);
  }}
  value={value}
  placeholder="Password"
  secureTextEntry
  autoCapitalize="none"
  textContentType={'password' || 'newPassword'}
/>
```
In Apple Developer Account I enabled Associated Domains as an entitlement in the mobile provision profile

In Firebase I created a Dynamic link (e.g. https://[appname].page.link

When I test the apple-app-site-association file by navigating in my brower (https://[appname].page.link/apple-app-site-association) I see

`{"applinks":{"apps":[],"details":[{"appID":"[DEV TEAM ID].com.[appname]","paths":["NOT /_/*","/*"]}]}}`

In XCode I add the Associated Domains cabability and point to the site which will host the apple-app-site-association file:

`webcredentials:[appname].page.link`

After these steps though, the autofill feature still does not work. I know I'm missing an important step but my Googling skills are coming up short today. Any ideas?

**Comment by siasjustin** [comment]
Any luck with this? I am trying to scope this out for a client.
